enzy_htp/mutation/api.py

```diff
@@ -367,8 +369,59 @@ def mutate_stru_with_tleap(
 
     return stru_cpy
 
+def mutate_stru_with_pymol(
+        stru: Structure,
+        mutant: List[Mutation],
+        in_place: bool = False,
+        ) -> Structure:
+    """mutate the {stru} to its {mutant} structure using PyMOL.
+    Args:
+        stru: the 'WT' structure
+        mutant: a list of Mutation() which describes a mutant to the 'WT'
+        in_place: if make the changes to the structure in-place 
+    Returns:
+        The mutated structure.
+    """
+
+    sp = PDBParser()
+    temp_file_name = fs.get_valid_temp_name("pymol_mutation_result.pdb")
```

Comment:
Please reference the way temp files are handled in mutate_stru_with_tleap() so that they are by default under a system.SCRATCH_DIR shared across enzy_htp. (avoid messy workdir like old EnzyHTP did)